src/urllib3/response.py

```diff
@@ -9,7 +9,10 @@
 import sys
 import typing
 import warnings
-import zlib
+try:
+    import zlib
+except ImportError:
```

Comment:
zlib is not currently available in wasi, so I guarded the import for now.
For all other systems there should not be a difference